app/bundles/LeadBundle/Segment/ContactSegmentFilterFactory.php

```diff
@@ -90,7 +90,7 @@ public function factorSegmentFilter(array $filter, array $batchLimiters = []): C
      *
      * @throws \Exception
      */
-    private function getQueryBuilderForFilter(FilterDecoratorInterface $decorator, ContactSegmentFilterCrate $contactSegmentFilterCrate)
+    private function getQueryBuilderForFilter(FilterDecoratorInterface $decorator, ContactSegmentFilterCrate $contactSegmentFilterCrate): ?object
```

Comment:
Can you please try?:
```suggestion
    private function getQueryBuilderForFilter(FilterDecoratorInterface $decorator, ContactSegmentFilterCrate $contactSegmentFilterCrate): ?FilterQueryBuilderInterface
```